compiler/rustc_mir_build/src/build/scope.rs

```diff
@@ -370,7 +370,7 @@ impl DropTree {
                     let terminator = TerminatorKind::Drop {
                         target: blocks[drop_data.1].unwrap(),
                         // The caller will handle this if needed.
-                        unwind: UnwindAction::Terminate,
+                        unwind: UnwindAction::Terminate(UnwindTerminateReason::InCleanup),
```

Comment:
I wasn't sure about the reason here since it is not obvious that we are in a cleanup block... but also, `Abi` seemed clearly wrong.

Reply:
I think `InCleanup` is the correct choice, certainly more so than `Abi` like you've said.